modulefiles/mpi/compilerName/compilerVersion/mpiName/mpiVersion/wgrib2/wgrib2.lua

```diff
@@ -0,0 +1,27 @@
+help([[
+]])
+
+local pkgName = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+local hierA        = hierarchyA(pkgNameVer,1)
+local compNameVer  = hierA[1]
+local compNameVerD = compNameVer:gsub("/","-")
```

Comment:
This is a module file in the MPI hierarchy, but the paths here are for the compiler hierarchy.

Reply:
For the CMake build of wgrib2 in `build_nceplibs`, which uses external NetCDF it was under MPI. If you want to build that it will fail to generate the module.

Reply:
Ok. Then, shouldn't these lines (8-10) be: 
```
local hierA        = hierarchyA(pkgNameVer,2)
local mpiNameVer   = hierA[1]
local compNameVer  = hierA[2]
local mpiNameVerD  = mpiNameVer:gsub("/","-")
local compNameVerD = compNameVer:gsub("/","-")
```